wbooks/src/app/screens/bookDetail/index.js

```diff
@@ -120,4 +102,20 @@ BookDetail.propTypes = {
   })
 }
 
-export default BookDetail;
+const mapStateToProps = (store) => ({
+  book: store.books.book,
+  books: store.books.books,
+  bookLoading: store.books.bookLoading,
+  bookFailed: store.books.bookFailed,
+  notAuthorized: store.books.notAuthorized,
```

Comment:
unAuthorized?

Reply:
i think this conveys the same meaning :face_with_head_bandage: , or why is unauthorized better?

Reply:
in fact, i'd "isAuthorized" , true or false.
What do you think?

Reply:
uhm, im not sure, the idea is that notAuthorized tells me if the books API did not accept my user.accessToken so then i could show an appropiate message. So i dont think isAuthorized would work because i'd have to initialize it with true, which is not true because i havent done any requests yet to know if my accessToken is valid. Im not convinced with "notAuthorized" but i cant think of another clear winner :stuck_out_tongue_closed_eyes:

Reply:
okay, noAuthiruzed so :smile: